x/monitor/types/genesis.go

```diff
@@ -33,17 +33,7 @@ func (gs GenesisState) Validate() error {
 	return nil
 }
 
-func (e EpochEndLightClient) Validate() error {
-	if e.BtcLightClientHeight == 0 {
```

Comment:
why remove these checks ?

Reply:
can happen that there's a epoch where there was no BTC lightclient height stored, so will default to 0